tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -491,10 +491,6 @@ def test_it_adds_learner_canvas_user_id_for_SpeedGrader_launches(self, sync):
             "group_set": None,
         }
 
-    @pytest.mark.usefixtures("canvas_groups_launch")
-    def test_present_if_group_assignment(self, sync):
```

Comment:
This test doesn't add much to 

`test_when_is_canvas`
`test_when_is_blackboard`